packages/gen-typescript-declarations/src/gen-ts.ts

```diff
@@ -74,6 +74,20 @@ export interface Config {
    * file.
    */
   autoImport?: {[modulePath: string]: string[]};
+
+  /**
+   * If true, outputs declarations in 'goog:' modules instead of using
+   * simple ES modules. This is a temporary hack to account for how modules are
+   * resolved for TypeScript inside google3. This is probably not at all useful
+   * for anyone but the Polymer team.
+   */
+  googModules?: boolean;
+
+  /**
+   * If true, does not log warnings detected when analyzing code,
+   * only diagnostics of Error severity.
+   */
+  hideWarnings?: boolean;
```

Comment:
Instead of a boolean, would it be nice to have the minimum error level here instead? E.g. the `true` now would mean `ERROR`.

Reply:
Maybe? It seems like the two reasonable options are either errors only, or errors and warnings, so it would make the API more awkward but no more expressive.